### DISCUSSION TITLE
Allow adding multiple website links

### DISCUSSION THREAD

**Discussion by smonolo** [community/community#110832] [Repositories]
### Select Topic Area

Product Feedback

### Body

Users can set a website link in the about section of the right sidebar. When working with a monorepo, it might happen that the repository includes code for multiple websites.

As a user, I want to be able to add multiple website links so I can show the diverse websites my code hosts.

The about section would be upgraded with a call-to-action that creates a new link input every time it's clicked, and one that removes an existing link.

**Comment by yCodeTech** [comment]
This would also benefit software that have published software to various registries. I need this for my vscode extension. I publish it to the main vscode marketplace, but I also now publish it to the open vsx registry. So I would prefer to list both links in the about section of the repo. So that users don't have to scroll down to the readme for the extra link.

**Comment by wavim** [comment]
Any updates here? An instance of my own is an npm package, where id like to show both the npm registry link and a demo github page's link.

**Comment by ackvf** [comment]
I am surprised this has such a small traction. I think about this every time I set up a repo. Having to put all the links in Description all the time feels wrong.

**Reply by smonolo** [reply]
It does feel wrong, but probably people just don't care enough about this. I don't think it would cost that much to be done though.
